@xen-orchestra/build-template/src/index.js

```diff
@@ -0,0 +1,21 @@
+import { escapeRegExp } from 'lodash'
+
+const compareLengthDesc = (a, b) => b.length - a.length
+
+export default (pattern, rules) => {
```

Comment:
The more I think about it, the more I prefer named export over default export.

I think we should name it `compileTemplate`.